sound/soc/sof/Kconfig

```diff
@@ -196,6 +196,7 @@ config SND_SOC_SOF_DEBUG_ENABLE_FIRMWARE_TRACE
 
 config SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST
 	tristate "SOF enable IPC flood test"
+	depends on SND_SOC_SOF
```

Comment:
typo in commit message: dependency

I am actually unsure if this commit message is correct, it seems that what is done here is only related to the client, as written by @ujfalusi below:

All config which select SND_SOC_SOF_CLIENT should depend on SND_SOC_SOF to make sure that they will follow the core's n/m/y